l10n_ec_ote/README.rst

```diff
@@ -0,0 +1,33 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Bug Tracker
```

Comment:
You should have kept all the previous part of the template and provide a description of what the modules does, how to use it and the link to runbot to test it.